packages/x-tree-view-pro/src/internals/plugins/useTreeViewItemsReordering/useTreeViewItemsReordering.ts

```diff
@@ -97,16 +99,18 @@ export const useTreeViewItemsReordering: TreeViewPlugin<UseTreeViewItemsReorderi
               ? targetItemIndex - 1
               : targetItemIndex,
         },
-        'reorder-below': targetItemMeta.expandable
-          ? null
-          : {
-              parentId: targetItemMeta.parentId,
-              index:
-                targetItemMeta.parentId === draggedItemMeta.parentId &&
-                targetItemIndex > draggedItemIndex
-                  ? targetItemIndex
-                  : targetItemIndex + 1,
-            },
+        // For the last element allow reordering to the bottom
+        'reorder-below':
+          !targetItemMeta.expandable || targetItemIndex + 1 === totalItems
```

Comment:
As I am looking into the diff, I feel the condition should be only applicable when `parentId` is `null`

Reply:
For me the same applies with nested elements.
In the video below, I can't move my item below `@mui/x-data-grid-premium` because it is expandable.

https://github.com/user-attachments/assets/82dc47f4-13e9-4be0-a6f9-6ed484383887

Reply:
yeah make sense, saw your changes, totally make sense. Sorry for the trouble you had to go through for the same. Will try to do better next time

Reply:
It was really not a critisism, I have the small advantage of knowing the codebase.
Community help is always greatly appreciated!